measures/advanced_import_gbxml/measure.rb

```diff
@@ -214,53 +214,71 @@ def run(model, runner, user_arguments)
-        ventilation = ventilation_element.text.to_f
-        ventilation = OpenStudio.convert(ventilation, "L/s", "cfm").get if ventilation_element.attributes['unit'] == "LPerSec"
-        ventilation_def[:ventilation_flow_per_space] = ventilation
+      # infiltration and ventilation data is in gbXML v7.03 AnalysisParameter element(s)
+      infiltration_def = {
+        infiltration_flow_per_space: nil,                 # cfm
+        infiltration_flow_per_space_area: nil,            # cfm/ft2
+        infiltration_flow_per_exterior_surface_area: nil, # cfm/ft2
+        infiltration_flow_per_exterior_wall_area: nil,    # cfm/ft2
+        infiltration_flow_air_changes_per_hour: nil       # 1/h
+      }
+      ventilation_def = {
+        ventilation_flow_per_person: 0.0,            # cfm
+        ventilation_flow_per_area: 0.0,              # cfm/ft2
+        ventilation_flow_per_space: 0.0,             # cfm
+        ventilation_flow_air_changes_per_hour: 0.0   # 1/h
+      }
+      if element.elements['AnalysisParameter'].size > 0
+        element.elements.each('AnalysisParameter') do |analysis_parameter|
+          next if analysis_parameter.elements['Name'].nil? && analysis_parameter.elements['ParameterValue'].nil?
+          name = analysis_parameter.elements['Name'].text
+          parameter_value = analysis_parameter.elements['ParameterValue'].text
+          unit = analysis_parameter.attributes['unit']
+          case name
+          when 'InfiltrationFlowPerArea'
+            case unit
+            when 'LPerSecPerSquareM'
+              infiltration = OpenStudio.convert(parameter_value.to_f, "L/s*m^2", "cfm/ft^2").get
+            else
+              infiltration = parameter_value.to_f
+            end
+            infiltration_def[:infiltration_flow_per_exterior_wall_area] = infiltration
+          when 'OAFlowMethod'
+            ventilation_def[:outdoor_airflow_method] = parameter_value
+          when 'OAFlowPerArea'
+            case unit
+            when 'LPerSecPerSquareM'
+              ventilation = OpenStudio.convert(parameter_value.to_f, "L/s*m^2", "cfm/ft^2").get
+            else
+              ventilation = parameter_value.to_f
+            end
+            ventilation_def[:ventilation_flow_per_area] = ventilation
+          when 'OAFlowPerPerson'
+            case unit
+            when 'LPerSec'
+              ventilation = OpenStudio.convert(parameter_value.to_f, "L/s", "cfm").get
+            else
+              ventilation = parameter_value.to_f
+            end
+            ventilation_def[:ventilation_flow_per_person] = ventilation
+          when 'OAFlowPerSpace'
+            case unit
+            when 'LPerSec'
+              ventilation = OpenStudio.convert(parameter_value.to_f, "L/s", "cfm").get
+            else
+              ventilation = parameter_value.to_f
+            end
+            ventilation_def[:ventilation_flow_per_space] = parameter_value.to_f
```

Comment:
Note: this parameter is currently unused: https://github.com/NREL/gbxml-to-openstudio/blob/f2857e6726fb9abf6dad5c16a9c4b8cc689eae33/measures/advanced_import_gbxml/resources/os_lib_adv_import.rb#L617

Reply:
I noticed that too and left per the original implementation. We can revisit later if needed.